src/main/java/TaskType.java

```diff
@@ -0,0 +1,5 @@
+public class TaskType extends Exception {
```

Comment:
The class name does not reflect well as an exception class. It sounded more like a class that objects can be instantiated instead.